android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -524,6 +525,19 @@ private void doMenuAction(int menuItemId) {
                 break;
             case R.id.nav_schedule:
                 replaceFragment(new ScheduleFragment(), R.string.menu_schedule);
+                boolean isLocal = SharedPreferencesUtil.getBoolean(getString(R.string.showLocalTime), false);
+                if(isLocal) {
+                    Calendar calendar = Calendar.getInstance(TimeZone.getTimeZone("GMT"),
+                            Locale.getDefault());
+                    Date currentLocalTime = calendar.getTime();
+                    DateFormat date = new SimpleDateFormat("z", Locale.getDefault());
+                    String localTime = date.format(currentLocalTime);
+                    getSupportActionBar().setTitle(getString(R.string.menu_schedule)+" ("+ localTime +")");
```

Comment:
Please use DateUtils for all date/time related operations.

The project has TheeTenABP, so all Java 7 date operations are deprecated and sholdn't be used

Reply:
Sorry for not being active because of end-semester. I will send the updated PR soon!